content/en/continuous_integration/setup_tests/ruby.md

```diff
@@ -377,7 +377,22 @@ DD_ENV=ci bundle exec rake spec
 
 {{% /tab %}}
 {{< /tabs >}}
+  
+### Adding custom tags to tests
 
+You can add custom tags to your tests by using the current active span:
+
+```ruby
+require 'ddtrace'
+
+# inside your test
+Datadog::Tracing.active_span&.set_tag('test_owner', 'my_team')
+# test continues normally
+# ...
+```
+
+To create filters or `group by` fields for these tags, you must first create facets. For more information about custom instrumentation, see the [Ruby Custom Instrumentation documentation][5].
```

Comment:
```suggestion
To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][5] section of the Ruby custom instrumentation documentation.
```

Reply:
Thanks for the review! These changes should be on all my 4 PRs now:

https://github.com/DataDog/documentation/pull/15099 (this one)
https://github.com/DataDog/documentation/pull/15097
https://github.com/DataDog/documentation/pull/15119
https://github.com/DataDog/documentation/pull/15120